example/resource-manager/10-pod-scheduler-name-mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Nit: Can you rename the file so that `mutatingwebhookconfiguration` comes first? (`10-mutatingwebhookconfiguration-...yaml`)